packages/health/lib/src/data_types.dart

```diff
@@ -254,6 +260,7 @@ enum HealthDataUnit {
 
   // Other units
   BEATS_PER_MINUTE,
+  RESPIRATIONS_PER_MINUTE,
```

Comment:
```suggestion
  COUNT_PER_MINUTE,
```
all future /minute types could make use of it then. Just my opinion, I don't own or work on this repository

Reply:
I followed the heart rate pattern

Reply:
yeah, mainly wanted to raise the discussion here. The heart rate is something that cannot be changed without breaking compatibility that's why I thought this might be worth a thought

Reply:
For now we have so few data types with this that I think it is okay to have some overlap, especially since the data type can be used to display additional context as in the example app. If the number of similar data types grows, this topic should be revisited.